ReactWindows/ReactNative.Bridge/ReflectionReactDelegateFactory.cs

```diff
@@ -57,7 +56,7 @@ private IList<Extractor> CreateExtractors(INativeModule module, MethodInfo metho
         private Extractor CreateExtractor(Type type, string moduleName, string methodName)
         {
             var exceptionFormat =
-                Invariant($"Error extracting argument for module '{moduleName}' method '{methodName}' at index '{{0}}'.");
+                $"Error extracting argument for module '{moduleName}' method '{methodName}' at index '{{0}}'.";
```

Comment:
This syntax unfortunately compiles to string.Format("...") without IFormatProvider.  Can we instead change all these to use the verbose string.Format(CultureInfo.InvariantCulture, "... {0} ... {1} ...", args)?